frontend/sentry.client.config.js

```diff
@@ -0,0 +1,19 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs"
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN
+
+Sentry.init({
+  dsn:
+    SENTRY_DSN ||
+    "https://b512f563700847e787e978ae1c15133c@o467221.ingest.sentry.io/6610580",
```

Comment:
That DSN is production one?

Reply:
I only see one instance in sentry, not sure if there are separate ones for production?

Reply:
My point is, we probably shouldn't make this URL public, as it's prone to abuse. I will need to land a secret `SENTRY_DSN` in CI/CD.

Reply:
Yes, I also thought about removing this, but I don't think it's private information, as it will end up in the client and you will be able to see, that your browser is sending data there, if you really want to find out.